### ISSUE TITLE
Search Engine Optimization - 301 Redirect Resolution

### ISSUE THREAD

**Issue by KonradMiles** [GSA/idmanagement.gov#578]
About the Issue:

A 301 redirect is used when a user or search engine lands on a URL with content that has been permanently moved to a new URL. The user is then redirected there. Where possible, redirects should be avoided as they slow down page load times and lead to loss of link equity. 

How to Fix:

Check URLS with 301 redirects and where possible, update them to point directly to the correct URL destination.

**Comment by claytonjbarnette** [comment]
@KonradMiles @SuGhadiali @id2win @TheInfinityBeyonder 

### The following addresses have been redirected from the Excel file. 

- /approved-products-list-apl/
- /governance/fpkiaudit/
- /fpkima/
- /fpkipa/
- /piv-i-for-issuers/
- /x509certpolicies/
